grammarinator/process.py

```diff
@@ -378,7 +378,12 @@ def chars_from_set(node):
 
         if node.STRING_LITERAL():
             assert len(str(node.STRING_LITERAL())) > 2, 'Negated string literal must not be empty.'
-            first_char = ord(str(node.STRING_LITERAL())[1])
+            # unescape '
+            c = node.STRING_LITERAL().symbol.text[1:-1].replace("\\'", "'")
```

Comment:
I guess `\\`, `\r`, `\n`, `\t` should be unescaped as well (and other less common characters), see https://github.com/antlr/antlr4/blob/master/tool/src/org/antlr/v4/parse/ANTLRLexer.g#L622